src/beast/beast/wsproto/socket.h

```diff
@@ -307,12 +348,12 @@ class socket : public detail::socket_base
     void
     accept(error_code& ec);
 
-    /** Start reading and responding to a WebSocket HTTP Upgrade request.
-
+    /// Start reading and responding to a WebSocket HTTP Upgrade request.
+    /*
```

Comment:
Not intentional. But even after this is merged I will make a few more passes through every header and tidy up the comments.